app/actions/definitions/documents.tsx

```diff
@@ -108,6 +110,63 @@ export const unstarDocument = createAction({
   },
 });
 
+export const subscribeDocument = createAction({
+  name: ({ t }) => t("Subscribe"),
+  section: DocumentSection,
+  // TODO: Use other icon.
+  icon: <BookmarkedIcon />,
```

Comment:
I have some icons in the pipeline for you to use, these are good placeholders 👍

Reply:
Upgrade `outline-icons` to `v1.44.0`, it includes `SubscribeIcon` and `UnsubscribeIcon`

Reply:
They look great! Checkout latest commit if you want to test them out. Dropdown menu should work.